src/main/java/com/gruzini/tennistico/events/RejectScoreEvent.java

```diff
@@ -0,0 +1,19 @@
+package com.gruzini.tennistico.events;
+
+import lombok.Getter;
+import org.springframework.context.ApplicationEvent;
+
+@Getter
+public class RejectScoreEvent extends ApplicationEvent implements ConfirmableEvent{
```

Comment:
Co robi confirmableEvent? To jakiś nasz interface?

Reply:
to jest nasz interfejs, umożliwia wydobycie id z każdej notyfikacji tego typu i wygaszenie buttona. Więc jeżeli chcesz żeby odpalenie eventu wygaszało buttona to po prostu implementujesz ten iterfejs i tyle